ajax.php

```diff
@@ -0,0 +1,140 @@
+<?php
+
+$dataType = $_GET["dataType"];
```

Comment:
tego pliku na razie nie komentuje, bo dla ciebie jako programisty javascript nie ma to wiekszego znaczenia. Jest tu mase rzeczy ktore sie powinno zrobic lepiej ale wole sie skupic na js skoro tego masz sie nauczyc :)